src/data/nat/fib.lean

```diff
@@ -82,6 +82,10 @@ lemma fib_le_fib_succ {n : ℕ} : fib n ≤ fib (n + 1) := by { cases n; simp [f
 @[mono] lemma fib_mono : monotone fib :=
 monotone_of_monotone_nat $ λ _, fib_le_fib_succ
 
+/-- If `2 ≤ n`, `fib` is strictly monotone. -/
+lemma fib_add_two_strict_mono : strict_mono (λ n, fib (n + 2)) := begin
```

Comment:
You'll need to delete the `begin` as well
```suggestion
lemma fib_add_two_strict_mono : strict_mono (λ n, fib (n + 2)) :=
```

Reply:
The one time I trust clicking buttons in a GitHub UI :)... Fixed.